dev-lang/luajit/luajit-2.0.4.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI="6"
```

Comment:
revbump this ebuild please (using `ebump -C` to all unstable keywords), and dont quote EAPI